src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/BasicFields.drl

```diff
@@ -44,6 +44,7 @@ rule "Copy basic fields and agenda controller"
         workloadInventoryReport.setVersion(vmWorkloadInventoryModel.getVersion());
         workloadInventoryReport.setHost_name(vmWorkloadInventoryModel.getHost_name());
         workloadInventoryReport.setCreationDate(vmWorkloadInventoryModel.getScanRunDate());
+        workloadInventoryReport.setInsightsEnabled(false);
```

Comment:
@m-brophy this line is inside the rule `"Copy basic fields and agenda controller"` and I think we should not set default values there since it should be used just to Copy but not other purposes... we can move this line to `reasonableDefaults.drl` or another option is to use a primitive boolean type (default value is `false`) in `WorkloadInventoryReportModel.java` so we have `private boolean insightsEnabled;` instead of `private Boolean insightsEnabled;`. There might be other options though, wdyt?